README.md

```diff
@@ -1 +1,48 @@
-# Leaderboard-list
\ No newline at end of file
+# Leaderboard-list
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is a leaderboard app where players can submit their scores. All the data will be preserved by the Leaderboard API service. This project has built with HTML, CSS, JavaScript, Webpack and Leaderboard API Service.
+
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- Webpack
+
+## Live Demo (if available)
+
+[Will be available soon]( )
+
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Web browser
+- Code editor
+
+### Setup
+```bash
+git clone (https://github.com/Masuduzzaman811/Leaderboard-list.git)
+cd Leaderboard-list
+```
+
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [Masuduzzaman811](https://github.com/Masuduzzaman811)
+- Twitter: [@msd_811](https://twitter.com/msd_811)
+- LinkedIn: [Masuduzzaman Mohammad](https://www.linkedin.com/in/msd811/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] I would also recommend adding the `Acknowledgments` and the issue link in the `Contributing` sections.
```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).


## Acknowledgments

- Microverse
```



- [ ] It will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications


- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

Reply:
Updated the README.md file as required. Thank you for the review.